wolfcrypt/benchmark/benchmark.c

```diff
@@ -7022,8 +7029,12 @@ static void bench_rsa_helper(int useDeviceID, RsaKey rsaKey[BENCH_MAX_PENDING],
                                           1, &times, ntimes, &pending)) {
                     #if !defined(WOLFSSL_RSA_VERIFY_INLINE) && \
                         !defined(WOLFSSL_RSA_PUBLIC_ONLY)
-                        ret = wc_RsaSSL_Verify(enc[i], idx, out[i],
+                        #if defined(WOLFSSL_MICROCHIP_TA100)
+                            ret = wc_RsaSSL_Verify(message, len, enc[i], rsaKeySz/8, &rsaKey[i]);
```

Comment:
Why would we change the behavior of the RSA verify API for one platform only? This should be handled in the API. If this is because of the ASN.1 encoding on the signature consider a different solution.

Reply:
Does the hardware really require the original message to perform a verification?

Reply:
Again, this is not correctly implemented. Please fix the internal TA100 calls for RSA. The test case should not need changed.

Reply:
Still outstanding, please address the feedback. Either with a comment on why is needed or a change in code.